ecosystem/sep-0031.md

```diff
@@ -137,8 +140,19 @@ The response should be a JSON object like:
                   ]
                }
             }
+         },
+         "fee_types": {
+            "cash_pickup": {
```

Comment:
❓ Are these fee types one-to-one with the values in the `choices` list? Or are they different?